src/main/java/minicraft/entity/furniture/KnightStatue.java

```diff
@@ -0,0 +1,95 @@
+package minicraft.entity.furniture;
+
+import minicraft.core.Game;
+import minicraft.core.io.Localization;
+import minicraft.entity.Direction;
+import minicraft.entity.mob.ObsidianKnight;
+import minicraft.entity.mob.Player;
+import minicraft.gfx.Screen;
+import minicraft.gfx.SpriteLinker;
+import minicraft.item.Item;
+
+public class KnightStatue extends Furniture {
+
+	private boolean touched = false; // second warn
+	private boolean tapped = false; // first warn
+	private boolean awoken = false; // awaking
+	private boolean touched1; // second warn spam failsafe
+	private boolean tapped1; // first warn spam failsafe;
+	public int obkHealth;
+	public static boolean active;
+
+	public KnightStatue(int health) {
+		super("KnightStatue", new SpriteLinker.LinkedSprite(SpriteLinker.SpriteType.Entity, "knight_statue"), new SpriteLinker.LinkedSprite(SpriteLinker.SpriteType.Item, "knight_statue"), 3, 2);
+		obkHealth = health;
+		ObsidianKnight.active = false;
+		ObsidianKnight.failed = false;
+		touched = false;
+		tapped = false;
+		awoken = false;
+		touched1 = false;
+		tapped1 = false;
+		active = true;
+	}
+
+	@Override
+	public void tick() {
+		super.tick();
+
+		if (!ObsidianKnight.active) {
+			if (tapped && !tapped1) {
+				// Warn the player
+				Game.notifications.add(Localization.getLocalized("minicraft.notifications.statue_tapped"));
+				tapped1 = true;
+			}
+			if (touched && !touched1) {
+				// Warn the player again
+				Game.notifications.add(Localization.getLocalized("minicraft.notifications.statue_touched"));
+				touched1 = true;
+			}
```

Comment:
It started as one Boolean and was added to. I’m a bit confused on how to use a integer value for this. I feel Booleans are better here because it makes it less likely to cause errors